README.rst

```diff
@@ -158,7 +158,7 @@ The minimal dependencies are::
 
 where ``Cython`` is only needed to benefit from the compiled accelerated simulator.
 The simulator itself has a pure Python counterpart, so that this requirement could
-be ignored, but is much slower.
+be ignored, but is much slower. For the Cython implementation to work, the user should have Eigen installed. 
```

Comment:
Actually, if you check the repo with the `--recursive` option, it should pull the eigen library too. I don't think the note is needed. This is covered in `CONTRIBUTING.rst`.